packages/oid4vci-holder/src/types/IOID4VCIHolder.ts

```diff
@@ -288,8 +288,7 @@ export type ErrorDetails = {
 export enum RequestType {
   OPENID_INITIATE_ISSUANCE = 'openid-initiate-issuance',
   OPENID_CREDENTIAL_OFFER = 'openid-credential-offer',
-  HTTPS = 'https',
-  HTTP = 'http',
+  URL = 'http',
```

Comment:
Because I'm following the flow of the existing code which checks url/uri starts with a specific scheme. In this case if I use the key URL it will match both http and https.

Reply:
Don't like that at all, but will change it at a later point I guess.